src/main/fc/config.h

```diff
@@ -33,7 +33,7 @@
 
 typedef enum {
     FEATURE_RX_PPM = 1 << 0,
-    //FEATURE_VBAT = 1 << 1,
+    //UNUSED = 1 << 1,
```

Comment:
I'd leave the commented out names in here, or entirely remove the lines. Having commented out lines that say `UNUSED` conveys no useful information.